src/main/java/pocketpal/frontend/constants/MessageConstants.java

```diff
@@ -49,20 +49,27 @@ public final class MessageConstants {
     public static final String MESSAGE_INVALID_COMMAND = "Please enter a valid command!";
     public static final String MESSAGE_INVALID_CATEGORY = "Please specify a valid category!";
     public static final String MESSAGE_INVALID_DESCRIPTION = "Description can only contain letters and numbers!";
+
     public static final String MESSAGE_INVALID_ID = "Please enter an integer from 1 to 2147483647.";
     public static final String MESSAGE_INVALID_AMOUNT = "Please enter a valid amount!" + NEWLINE
             + "Value should be between 0.01 and 999999999.99";
     public static final String MESSAGE_INVALID_AMOUNT_RANGE = "Please specify a valid range!" + NEWLINE
             + "Values should be between 0 and 1000000000";
+    public static final String MESSAGE_UNKNOWN_OPTION = "Unknown option: ";
+    public static final String MESSAGE_MISSING_REQUIRED_OPTION = "Missing required options: ";
+    public static final String MESSAGE_MISSING_OPTION_EDIT = "Please specify at least one option!";
+    public static final String MESSAGE_MISSING_OPTION_ARG = "Missing argument for option: ";
     public static final String MESSAGE_MISSING_ARGS_ADD = "Please specify the description, category and price!";
     public static final String MESSAGE_MISSING_PRICE_ADD = "Please specify the price using the '-p' flag!";
     public static final String MESSAGE_MISSING_DESCRIPTION_ADD = "Please specify the description using the '-d' flag!";
     public static final String MESSAGE_MISSING_CATEGORY_ADD = "Please specify the category using the '-c' flag!";
```

Comment:
Are these constants meant to be used? It seems like there is no code referencing these constants.